api/api.go

```diff
@@ -45,42 +40,23 @@ func CreateServer(elrondProxyFacade ElrondProxyHandler, port int) (*http.Server,
 	return httpServer, nil
 }
 
-func registerRoutes(ws *gin.Engine, elrondProxyFacade ElrondProxyHandler) {
-	addressRoutes := ws.Group("/address")
-	addressRoutes.Use(WithElrondProxyFacade(elrondProxyFacade))
-	address.Routes(addressRoutes)
-
-	txRoutes := ws.Group("/transaction")
-	txRoutes.Use(WithElrondProxyFacade(elrondProxyFacade))
-	transaction.Routes(txRoutes)
-
-	getValuesRoutes := ws.Group("/vm-values")
-	getValuesRoutes.Use(WithElrondProxyFacade(elrondProxyFacade))
-	vmValues.Routes(getValuesRoutes)
-
-	networkRoutes := ws.Group("/network")
-	networkRoutes.Use(WithElrondProxyFacade(elrondProxyFacade))
-	network.Routes(networkRoutes)
-
-	nodeRoutes := ws.Group("/node")
-	nodeRoutes.Use(WithElrondProxyFacade(elrondProxyFacade))
-	node.Routes(nodeRoutes)
-
-	validatorRoutes := ws.Group("/validator")
-	validatorRoutes.Use(WithElrondProxyFacade(elrondProxyFacade))
-	valStats.Routes(validatorRoutes)
+func registerRoutes(ws *gin.Engine, versionManager data.VersionManagerHandler) error {
+	versionsMap, err := versionManager.GetAllVersions()
+	if err != nil {
+		return err
+	}
 
-	blockAtlasRoutes := ws.Group("/block-atlas")
-	blockAtlasRoutes.Use(WithElrondProxyFacade(elrondProxyFacade))
-	blockatlas.Routes(blockAtlasRoutes)
+	for version, versionData := range versionsMap {
```

Comment:
I think the current naming is better because `VersionData` only holds the specific components for each version and it doesn't have any functions.
However, renamed `VersionManager` to `VersionsRegistry`